packages/mobile/src/account/saga.ts

```diff
@@ -112,6 +114,30 @@ export function* watchDailyLimit() {
   }
 }
 
+export function* watchKycStatus() {
+  const account = yield call(getWalletAddress)
+  const channel = yield call(kycStatusChannel, account)
```

Comment:
I think there's a bug here-- to get the user's kyc from firebase you'll need their MTW address, not their EOA address

https://docs.celo.org/celo-codebase/protocol/identity/valora-accounts

Reply:
this also means users need an MTW set up already (which happens during phone verification) to access the feature. If that comes as a surprise we should discuss further to make sure the requirement is accounted for in our designs and GH issues.

Reply:
It looks like if the user doesnt have phone number verified, there's a `unverifiedMtwAddress` we can get from Komenci